.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "presets": [
+      "@parcel/babel-preset-env",
+      "@babel/preset-react"
+  ],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties",
```

Comment:
This seems to be part of preset env, it might just need to be enabled https://babeljs.io/blog/2020/05/25/7.10.0